packages/fluentui/react-northstar/src/components/Pill/Pills.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import { Accessibility, pillsBehavior, PillsBehaviorProps } from '@fluentui/accessibility';
+import {
+  ComponentWithAs,
+  getElementType,
+  useUnhandledProps,
+  useAccessibility,
+  useFluentContext,
+  useStyles,
+  useTelemetry,
+} from '@fluentui/react-bindings';
+import {
+  UIComponentProps,
+  ChildrenComponentProps,
+  ContentComponentProps,
+  commonPropTypes,
+  rtlTextContainer,
+} from '../../utils';
+
+import { FluentComponentStaticProps } from '../../types';
+import { PillsContextProvider, PillsContextValue } from './pillContext';
+
+export interface PillsProps extends UIComponentProps, ChildrenComponentProps, ContentComponentProps {
+  /**
+   * Accessibility behavior if overridden by the user.
+   */
+  accessibility?: Accessibility<PillsBehaviorProps>;
+}
+
+export type PillsStylesProps = never;
+export const PillsClassName = 'ui-pills';
+
+/**
+ * A Pills can be used as container for Pill.
+ */
+export const Pills: ComponentWithAs<'div', PillsProps> & FluentComponentStaticProps<PillsProps> = props => {
+  const context = useFluentContext();
+  const { setStart, setEnd } = useTelemetry(Pills.displayName, context.telemetry);
+  setStart();
+
+  const { accessibility, children, className, design, styles, variables } = props;
+
+  const getA11Props = useAccessibility(accessibility, {
+    debugName: Pills.displayName,
+    rtl: context.rtl,
+  });
+
+  const { classes } = useStyles<PillsStylesProps>(Pills.displayName, {
+    className: PillsClassName,
+    mapPropsToInlineStyles: () => ({ className, design, styles, variables }),
+    rtl: context.rtl,
+  });
+
+  const ElementType = getElementType(props);
+  const unhandledProps = useUnhandledProps(Pills.handledProps, props);
+
+  const childProps: PillsContextValue = {
+    role: 'option',
+  };
```

Comment:
you should memo the context value, since object references are always recreated and will always result a rerender of all the provider children

Reply:
Done